src/nbl/asset/interchange/CImageWriterTGA.cpp

```diff
@@ -134,7 +134,7 @@ bool CImageWriterTGA::writeAsset(system::IFile* _file, const SAssetWriteParams&
 	int32_t row_size = ((imageHeader.PixelDepth / 8) * imageHeader.ImageWidth);
 
 	// allocate a row do translate data into
-	auto rowPointerBuffer = core::make_smart_refctd_ptr<ICPUBuffer>(row_size);
+	auto rowPointerBuffer = ICPUBuffer::create({ (size_t)row_size });
```

Comment:
if you're putting `size_t` because compiler screams warnings at you, use a `static_cast` in place of a C-style cast